views/stylesheet.css

```diff
@@ -0,0 +1,59 @@
+.calculator-container {
+  width: 232px;
+  height: 323px;
+  border-radius: 15px;
+  position: absolute;
```

Comment:
@teqnickels this is an interesting solution to the center/center problem. This works in some ways but fails in some others. This does in fact center/center the calculator but `position: absolute` causes the calculator to float above the page. This causes the calculator to not take up any space. So you'll notice that the body element on your page has 0 height. 

TL;DR: Using `position: absolute` in combination with `margin: auto` is an interesting trick but I strongly encourage you to look into flexbox.